pulseaudio-control.bash

```diff
@@ -55,21 +55,21 @@ function getCurNode() {
     local curNodeName
 
     curNodeName=$(pactl info | awk "/Default S${SINK_OR_SOURCE}: / {print \$3}")
-    curNode=$(pactl list s${SINK_OR_SOURCE}s | grep -B 4 -E "Name: $curNodeName\$" | sed -nE "s/^S${SINK_OR_SOURCE} #([0-9]+)$/\1/p")
+    curNode=$(pactl list s"${SINK_OR_SOURCE}"s | grep -B 4 -E "Name: $curNodeName\$" | sed -nE "s/^S${SINK_OR_SOURCE} #([0-9]+)$/\1/p")
```

Comment:
How about `"s${SINK_OR_SOURCE}s"`? That looks more natural here. This seems to happen with every change in this PR (other than typos).

Reply:
I've made those changes and I agree that it does look more natural. Please double check to make sure I'm not missing something.

Thank you.